### ISSUE TITLE
Prevent Spawning of Guardians, Zombie Horses and Skeleton Horses

### ISSUE THREAD

**Issue by InterplanetaryEngineer** [cuberite/cuberite#3005]
I found all three of them in my river, where they certainly don't belong. Zombie/Skeleton Horses don't occur naturally at all in 1.8, and Guardians should only occur in and around Ocean Monuments (wich someone should finally implement, by the way, see #1423 ).

**Comment by LogicParrot** [comment]
Did you see the Skeleton Horses only after world teleportation? Would logging out and back in turn them into another time of animal?

**Comment by tigerw** [comment]
I remember intentionally allowing skeleton and zombie horses to spawn because pfft, Mojang have no power over us!

**Comment by ShrewdSpirit** [comment]
Also I could spawn a zombie horse with horse egg! I don't think this is natural :D

**Comment by jammet** [comment]
Yes, I've seen them around as well, in fresh, generated chunks, while traversing the world.

**Comment by Titaniumtown** [comment]
The spawning of guardians still happens for me, I also see random other mobs like bats, skeletons, zombies, and chickens just spawning in the middle of the ocean.
